query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryServiceImpl.java

```diff
@@ -16,14 +18,40 @@
 import org.hypertrace.core.query.service.api.QueryServiceGrpc;
 import org.hypertrace.core.query.service.api.ResultSetChunk;
 import org.hypertrace.core.query.service.validation.QueryValidator;
+import org.hypertrace.core.serviceframework.metrics.PlatformMetricsRegistry;
 
 @Singleton
 @Slf4j
 class QueryServiceImpl extends QueryServiceGrpc.QueryServiceImplBase {
+
   private final RequestHandlerSelector handlerSelector;
   private final QueryTransformationPipeline queryTransformationPipeline;
   private final QueryValidator queryValidator;
 
+  private Counter serviceResponseErrorCounter;
+  private Counter serviceResponseSuccessCounter;
+  private static final String ERROR_COUNTER_NAME = "hypertrace.query-service.response.errors";
```

Comment:
See the comment on your other PR - https://github.com/hypertrace/gateway-service/pull/128/files#r854786951
Shall we go with?
`hypertrace.query.service.request.status {"error"="true"}`
`hypertrace.query.service.request.status {"error"="false"}`

Reply:
Please

Reply:
Yes, making this change.